job_templates/puppet_run_once_-_ansible_default.erb

```diff
@@ -0,0 +1,21 @@
+<%#
+name: Puppet Run Once - Ansible Default
+job_category: Puppet
+description_format: "%{action} host"
+snippet: false
+template_inputs:
+- name: puppet_options
+  required: false
+  input_type: user
+  options: Additional options to pass to Puppet
```

Comment:
this is not a valid option, this should have been description I guess, options should not be set as it's free text input